ingest/src/main/java/com/microsoft/azure/kusto/ingest/IngestionProperties.java

```diff
@@ -290,6 +290,17 @@ void validate() {
         }
     }
 
+    public void validateResultSetProperties() {
+        String givenDataFormat = this.getDataFormat();
+        if (givenDataFormat == null) {
+            this.setDataFormat(IngestionProperties.DataFormat.csv);
```

Comment:
I assume this is documented that CSV will be set by default if the user doesn't set it

Reply:
We know it is now, right?

Reply:
Now it's in the Javadoc (or once the newest PR is merged)